### ISSUE TITLE
Handle Field-level Markings

### ISSUE THREAD

**Issue by emmanvg** [oasis-open/cti-stix-elevator#84]
Currently the stix2elevator only supports global and object level markings. There is no support for field level markings. Also, no message is printed to indicate we encountered one.

Related to #13

**Comment by gtback** [comment]
How big of a priority is this? I agree we should at least print a warning/error message, but actually translating them seems like a difficult task.

**Comment by emmanvg** [comment]
I believe you can mention it your commit message without the need to close or create a new issue. In the commit message include "Added log message for Field-Level Markings. related to (hash-tag)84" (where hash-tag is the actual character). I don't know if adding it to the milestone would be relevant since it is not solving the actual problem.

**Comment by emmanvg** [comment]
I agree @gtback, the message should be enough for now. I just created an issue for bookkeeping. Up to now, there has not been a request to support it.

**Comment by rpiazza** [comment]
an issue is just something that needs to "get done" :-)

We want to be able to present to Preston a list of the work we intend to do so the elevator is ready for AIS

**Comment by emmanvg** [comment]
AIS makes use of global markings. Which is already supported by the elevator. Field-Level markings is a general case scenario you may encounter with markings.

**Comment by emmanvg** [comment]
The PR that provides the functionality can be added to the milestone.
